.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+# Contributing
```

Comment:
How is this file used? Thoughts on using a "see docs" stub like we had before? Helps reduce duplication of info and sources.

Reply:
Interface-wise, it gets linked to in this section...

(From the "New Issue" page)

![Screen Shot 2021-03-24 at 16 36 06](https://user-images.githubusercontent.com/647359/112347891-2d34a880-8cbf-11eb-86ad-a4ca83719dc6.png)

I'm equally happy with it either as a copy of the docs, or as a simple docs link.